File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BasePartyPCType.vb
Public Class BasePartyPCType
    Inherits BasePartyType
    Public Property Surname As String
    Public Property Forename As String
    Public Property DateOfBirth As Date
    Public Property DateOfBirthSpecified As Boolean
    Public Property Title As String
    Public Property MaritalStatusCode As MaritalStatusCodeType
    Public Property MaritalStatusCodeSpecified As Boolean
    Public Property GenderCode As String
    Public Property Initials As String
    Public Property OccupationCode As String
    Public Property EmployersBusinessCode As String
    Public Property EmploymentStatusCode As EmploymentStatusCodeType
    Public Property EmploymentStatusCodeSpecified As Boolean
    Public Property AlternativeId As String
    Public Property ClientDetail As BaseClientSharedDataType
    Public Property NationalityCode As String
    Public Property Salutation As String
    Public Property TradingName As String
    Public Property SecOccupationCode As String
    Public Property SecEmployersBusinessCode As String
    Public Property SecEmploymentStatusCode As EmploymentStatusCodeType
    Public Property SecEmploymentStatusCodeSpecified As Boolean
    Public Property AccommodationCode As String
    Public Property Lifestyle As List(Of BasePartyPCTypeLifestyle)
    Public Property TPS As Boolean
    Public Property TPSSpecified As Boolean
    Public Property MPS As Boolean
    Public Property MPSSpecified As Boolean
    Public Property EMPS As Boolean
    Public Property EMPSSpecified As Boolean
    Public Property Source As String
    Public Property PetOwner As Boolean
    Public Property PetOwnerSpecified As Boolean
End Class

File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BasePartyPCType.cs
using SSP.PureInsuranceRestAPIHandler.Enums;
using System.Collections.Generic;

namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BasePartyPCType : BasePartyType
    {
        public string Surname { get; set; } = string.Empty;
        public string Forename { get; set; } = string.Empty;
        public System.DateTime DateOfBirth { get; set; }
        public bool DateOfBirthSpecified { get; set; }
        public string Title { get; set; } = string.Empty;
        public MaritalStatusCodeType MaritalStatusCode { get; set; }
        public bool MaritalStatusCodeSpecified { get; set; }
        public string GenderCode { get; set; } = string.Empty;
        public string Initials { get; set; } = string.Empty;
        public string OccupationCode { get; set; } = string.Empty;
        public string EmployersBusinessCode { get; set; } = string.Empty;
        public EmploymentStatusCodeType EmploymentStatusCode { get; set; }
        public bool EmploymentStatusCodeSpecified { get; set; }
        public string AlternativeId { get; set; } = string.Empty;
        public BaseClientSharedDataType ClientDetail { get; set; } = null;
        public string TradingName { get; set; } = string.Empty;
        public string SecOccupationCode { get; set; } = string.Empty;
        public string SecEmployersBusinessCode { get; set; } = string.Empty;
        public EmploymentStatusCodeType SecEmploymentStatusCode { get; set; }
        public bool SecEmploymentStatusCodeSpecified { get; set; }
        public string NationalityCode { get; set; } = string.Empty;
        public string AccommodationCode { get; set; } = string.Empty;
        public System.Collections.Generic.List<BasePartyPCTypeLifestyle> Lifestyle { get; set; } = null;
        public string Salutation { get; set; } = string.Empty;
        public bool TPS { get; set; }
        public bool TPSSpecified { get; set; }
        public bool MPS { get; set; }
        public bool MPSSpecified { get; set; }
        public bool EMPS { get; set; }
        public bool EMPSSpecified { get; set; }
        public string Source { get; set; } = string.Empty;
        public bool PetOwner { get; set; }
        public bool PetOwnerSpecified { get; set; }
    }
}
